Wabbajack.App.Wpf/View Models/Installers/InstallerVM.cs

```diff
@@ -336,7 +336,7 @@ private IEnumerable<ErrorResponse> Validate()
 
         if (KnownFolders.IsInSpecialFolder(installPath) || KnownFolders.IsInSpecialFolder(downloadPath))
         {
-            yield return ErrorResponse.Fail("Can't install a modlist into Windows protected locations - such as Downloads,Documents etc, please make a new folder for the modlist.");
+            yield return ErrorResponse.Fail("Can't install into Windows locations such as Documents etc, please make a new folder for the modlist - C:\Wabbajack\ for example.");
```

Comment:
**THIS IS A BAD EXAMPLE !!**

Because with this users will wipe the recommended WJ location making WJ try to delete itself...

😅

Reply:
Oops I forgot what i was looking at, I thoguth for a moment this was the wabbajack.exe location error! sorry my bad

Reply:
not a big deal, just happy I caught it 😂